src/taipy/core/config/data_node_config.py

```diff
@@ -284,22 +324,23 @@ def _configure_default(storage_type: str, scope: Scope = _DEFAULT_SCOPE, **prope
             storage_type (str): The default storage type for all data node configurations.
                 The possible values are _"pickle"_ (the default value), _"csv"_, _"excel"_,
                 _"sql"_, _"mongo_collection"_, _"in_memory"_, _"json"_, _"parquet"_ or _"generic"_.
-            scope (Scope^): The default scope for all data node configurations.
+            scope (Optional[Scope^]): The default scope for all data node configurations.
                 The default value is `Scope.SCENARIO`.
             **properties (Dict[str, Any]): A keyworded variable length list of additional
                 arguments.
         Returns:
             `DataNodeConfig^`: The default data node configuration.
         """
         section = DataNodeConfig(_Config.DEFAULT_KEY, storage_type, scope, **properties)
-        Config._register(section)
+        Config._register_default(section)
```

Comment:
This is to differentiate with the generic configure methods, and to replace the "default" config instead of update like the generic one.

The tests are failing since we need an update on https://github.com/Avaiga/taipy-config/pull/47 as well.